src/views/DatasetManager/components/DatasetOverview/components/DatasetActions/DatasetActions.js

```diff
@@ -1,11 +1,28 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { FileCopy, AddCircle as AddCircleIcon, Share as ShareIcon } from '@mui/icons-material';
+import EditIcon from '@mui/icons-material/Edit';
+import { IconButton } from '@mui/material';
 import ButtonGroup from '@mui/material/ButtonGroup';
 import { DeleteDatasetButton } from '../../../DeleteDatasetButton/DeleteDatasetButton';
+import { RefreshDatasetButton } from '../RefreshDatasetButton/RefreshDatasetButton';
 
 export default function DatasetActions({ dataset }) {
   return (
     <ButtonGroup>
+      <RefreshDatasetButton dataset={dataset} />
+      <IconButton>
+        <EditIcon color="primary" />
+      </IconButton>
+      <IconButton>
+        <FileCopy color="primary" />
+      </IconButton>
+      <IconButton>
+        <AddCircleIcon color="primary" />
+      </IconButton>
+      <IconButton>
+        <ShareIcon color="primary" />
+      </IconButton>
```

Comment:
As discussed, we probably don't want to have fake buttons in our _main_ branch :smile: